ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -119,23 +119,33 @@
               >
                 <b>DELETED</b>
               </v-chip>
+              <v-chip
+                v-if="isMhrTransfer && showDeathCertificate && isRemovedHomeOwner(row.item)"
+                class="badge-delete ml-8 mt-2"
+                label x-small
+                color="#grey lighten-2"
+                text-color="$gray9"
+                data-test-id="owner-deceased-badge"
+              >
+                <b>DECEASED</b>
+              </v-chip>
```

Comment:
The only difference i see between the two badges is the chip content (ie Deleted vs Deceased) and the test-id. 
Perhaps we just template the content vs a whole new badge? 
ie `<b>{{ showDeathCertificate ? 'DECEASED' : 'DELETED' }}</b>`?

Side question though, have you looked at the spec, does it require a deleted or deceased action type to be passed in the payload or is it just assumed in this flow?

Reply:
oo good question, I'll check

Reply:
"Deleted owners (deceased) or surviving owners must be a OWNER_BUS or OWNER_IND partyType."
This is under the TRAND section in the spec and makes me assume the API takes the DELETED action type. Didn't see anything more specific. @doug-lovett are you able to confirm?

Reply:
The api has no such action for the owner in a deleteOwnerGroup for this registration. If the death certificate information is present the owner is deceased. If the the death certificate information is not provided for an owner in the deleteOwnerGroup, that owner must be in the addOwnerGroup as a surviving owner.

Reply:
The OWNER_BUS, OWNER_IND partyType rule means the group cannot have an EXECUTOR/ADMINISTRATOR/TRUSTEE.

Reply:
Cool, sounds good! 
So its really just some interchangeable text 👍